### ISSUE TITLE
Errno::ECONNREFUSED at /plannings Failed to open TCP connection to localhost:9090 (Connection refused - connect(2) for "localhost" port 9090)

### ISSUE THREAD

**Issue by meloBerati** [Mapotempo/mapotempo-web#11]
Errno::ECONNREFUSED at /plannings
Failed to open TCP connection to localhost:9090 (Connection refused - connect(2) for "localhost" port 9090)

![screenshot from 2018-04-15 17-34-44](https://user-images.githubusercontent.com/38402992/38780210-56a96614-40d3-11e8-9b09-c6b157ccb603.png)

**Comment by frodrigo** [comment]
You should configure a proper routing engine here:
https://github.com/Mapotempo/mapotempo-web/blob/44b1fa671eddec16cd6406899083ec97ab7fd209/db/seeds.rb#L15
Available routing engine are here :https://github.com/Mapotempo/mapotempo-web/tree/master/lib/routers

**Comment by meloBerati** [comment]
after configured the router engine it doesnt starts and gives the same error..
Any idea?

**Comment by ilstarno** [comment]
`

60 61 62 63 64 65 | else               response = (response && /json/.match(response.headers[:content_type]) && response.size > 1) ? JSON.parse(response) : nil               raise RouterError.new(result.message + (response && response['message'] ? ' - ' + response['message'] : ''))             end           }           if request != ''
-- | --


`

**Comment by ilstarno** [comment]
RouterError in PlanningsController#create

**Comment by ilstarno** [comment]
Extracted source (around line #62):

**Comment by meloBerati** [comment]
Errno::ECONNREFUSED in PlanningsController#create

Failed to open TCP connection to localhost:4899 (Connection refused - connect(2) for "localhost" port 4899)

      res = execute_without_capture_duration(args, &block)

**Comment by meloBerati** [comment]
RouterError in PlanningsController#create

**Comment by frodrigo** [comment]
Please, add more details if you want help.
What do you configure as router, and more detail about stack trace.

**Comment by johnGarcia123** [comment]
Yeah i cant configure it to..  there is no localhost:9090 adress on my pc..
Any advice?

**Comment by frodrigo** [comment]
localhost:9090 is the default routing engine. As already wrote you must adapt it to your setup, by deploy  an OSRM or a router wrapper.
